src/moment.module.ts

```diff
@@ -29,11 +30,17 @@ const ANGULAR_MOMENT_PIPES = [
   FromUtcPipe,
   LocalTimePipe,
   LocalePipe,
-  ParseZonePipe
+  ParseZonePipe,
+  IsBeforePipe,
+  IsAfterPipe
 ];
 
 @NgModule({
-  declarations: ANGULAR_MOMENT_PIPES,
-  exports: ANGULAR_MOMENT_PIPES
+  declarations: [
```

Comment:
What's the reason for this change?

Reply:
No reason other than my IDE screwed up the module file when adding the pipes as it injects the declarations into the `declarations: []` section of the module

Reply:
I can revert this as part of this PR if you prefer

Reply:
Please do

Reply:
Done

Reply:
Thank you!